src/OAuthgen/Provider.php

```diff
@@ -22,8 +22,8 @@ class Provider extends AbstractProvider
     /**
      * BASE_URL.
      */
-    protected $oauth_url = 'https://auth.oauthgen.com';
-    protected $graph_url = 'https://graph.oauthgen.com/api/v1';
+    protected $oauthUrl = 'https://auth.oauthgen.com';
```

Comment:
these could be protected consts instead of props i guess

Reply:
See 010d222075c4451018beadc743c05c5b3bdf5680

Reply:
@atymic I think the PR was merged before this commit

Reply:
oops 🙈 
Chuck in your next PR :)